requirements.txt

```diff
@@ -1,3 +1,4 @@
 pytest
 flake8
 attrs>=18.1.0
+mypy
```

Comment:
This clashes with #5 adding a mypy requirement too

Reply:
Yeah, just remembered about that. We could take this one first and then @manfredlift can rebase?

Reply:
... or merge

Reply:
I've already added this in my PR?